contollers/list.controller.js

```diff
@@ -0,0 +1,63 @@
+const List = require("../models/list.model");
+
+class ListController {
+  static async index(req, res) {
+    try {
+      res.send(await List.find({}));
+    } catch (err) {
+      console.log(err);
+      res.sendStatus(500);
+    }
+  }
+
+  static async create(req, res) {
+    try {
+      let list = await List.create({
+        title: req.body.title,
+        description: req.body.description,
+        userId: req.user.id,
+      });
+
+      res.send(list);
+    } catch (err) {
+      console.log(err);
+      res.sendStatus(500);
+    }
+  }
+
+  static async show(req, res) {
+    try {
+      const result = await List.findById(req.params.id);
+      if (result) res.send(result);
+      else res.send("No result");
+    } catch (err) {
+      console.log(err);
+      res.sendStatus(500);
+    }
+  }
+
+  static async update(req, res) {
+    try {
+      updetedValues = {
+        title: req.body.title,
+        discription: req.body.discription,
+      };
+      await List.findByIdAndUpdate(req.params.id, updetedValues);
+      res.send(await List.findById(req.params.id));
```

Comment:
You're making 2 DB requests here, you can use the findByIdAndUpdate to fetch the updated value in a single call